### DISCUSSION TITLE
Poor support for Right To Left languages

### DISCUSSION THREAD

**Discussion by hamiddd1980** [dotnet/maui#14856] [General]
Hello and thank you MAUI team for your efforts and hard work. But I want to complain of poor support for Right To Left languages in MAUI product especially **about Graphical objects( like Line,Rectangle, Shapes ...) , Clip property** and ... .
The problem has existed from Xamarin.Forms and it seems MAUI team is not going to consider it (full support for Right To Left layout) as a key feature in their Production (perhaps because of less popularity of Right to Left languages).
But missing full support for Right To Left cultures (like Hebrew) is disappointing  and discouraging and in fact, prevents users from developing multilingual  business applications. 
Thank you.

**Comment by Xyncgas** [comment]
Maui.blazor can use web to deal with this, that's one way..
